email/1.3/modules/ROOT/edits/email-trigger.adoc

```diff
@@ -0,0 +1,219 @@
+= Trigger a Flow When Receiving a New Email Examples - Mule 4
+:page-aliases: connectors::email/email-trigger.adoc
+
+The following examples illustrate how you configure the Anypoint Connector for Email (Email Connector) source listeners *On New Email - IMAP* and *On New Email - POP3* to identify and retrieve unread email messages, initiating a flow.
+
+== Configure Delete After Retrieve Parameter
+
+Email Connector *On New Email - IMAP* and *On New Email - POP3* sources support the *Delete after retrieve* parameter that enables the deletion of retrieved emails from the mailbox folder after the operation process is complete. By deleting processed emails, you can ensure that the next processed emails are new ones. Because the POP3 protocol doesn't support watermarking, the *Delete after retrieve* parameter is commonly used with this protocol. +
+The parameter is disabled by default to avoid deleting emails by mistake.
```

Comment:
I don't understand this. So there are circumstances under which a user would want both processed and unprocessed email in the mailbox? How could it be a mistake to delete email after it no longer serves its purpose?

Reply:
(I know connector docs don't use them, but I would normally suggest a graphic depiction of this process of incoming email, retrieved email, operation processing, deleting email, moving to new incoming email....)